bin/update-en-comments

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use CodeIgniter\CLI\CLI;
+
+helper('filesystem');
+
+if ($argc !== 2) {
+    CLI::error('Please specify a locale.');
+
+    exit(1);
+}
+
+$locale = $argv[1];
+
+$langDir = __DIR__ . '/../Language/' . $locale;
+if (! is_dir($langDir)) {
+    CLI::error('No such directory: "' . $langDir . '"');
+
+    exit(1);
+}
+
+$enDir = VENDORPATH . 'codeigniter4/CodeIgniter4/system/Language/en';
+if (! is_dir($enDir)) {
+    CLI::error('No "Language/en" directory. Please run "composer update".');
+
+    exit(1);
+}
+
+$files = get_filenames(
+    $langDir,
+    true,
+    false,
+    false
+);
+
+$enFiles = get_filenames(
+    $enDir,
+    true,
+    false,
+    false
+);
+
+foreach ($enFiles as $enFile) {
+    $langFile = $langDir . '/' . substr($enFile, strlen($enDir) + 1);
+    if (! is_file($langFile)) {
+        CLI::error('No such file: "' . $langFile . '"');
+
+        continue;
+    }
```

Comment:
For readability of the filename in the output later, I suggest using `realpath()`
```suggestion
    $langFile = $langDir . '/' . substr($enFile, strlen($enDir) + 1);
    $temp     = realpath($langFile);
    
    if (! is_file($temp)) {
        CLI::error('No such file: "' . $langFile . '"');

        continue;
    }
    
    $langFile = $temp;
```

Reply:
I updated the code.